app/src/main/java/com/guard/security/crc/crc_guard_app/webview/mWebClient.java

```diff
@@ -0,0 +1,24 @@
+package com.guard.security.crc.crc_guard_app.webview;
+
+import android.graphics.Bitmap;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+public class mWebClient  extends WebViewClient {
+    //Si está en 0 significa que la pagina no se ha terminado de cargar
+    //Al pasar a 1 ya está cargada
+    private String Carga = "0";
+
+    @Override
+    public void onPageStarted(WebView view, String url, Bitmap favicon) {
+        if (Carga != "0"){
```

Comment:
Our static analysis engine detects that your pull request includes a change from a Comparison of String objects using `==` or `!=`. According to [this description](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq), unless both strings are either constants in a source file, or have been interned using the `String.intern()` method, the same string value may be represented by two different String objects. Consider using the `equals(Object)` method instead.